docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1495,3 +1495,115 @@ Converts a string to its uppercase equivalent.
   }
 }
 --------------------------------------------------
+
+[[dot-expand-processor]]
+=== Dot Expander Processor
+
+Expands a field with dots into an object field. This processor allows fields
+with dots in the name to be accessible by other processors in the pipeline.
+Otherwise these fields can't accessed by any processor.
```

Comment:
as a side note, should we link to this section here? https://www.elastic.co/guide/en/elasticsearch/reference/master/accessing-data-in-pipelines.html